core/server/models/base/index.js

```diff
@@ -882,6 +882,14 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
             });
         };
 
+        const removeInvisibleUnicode = (str) => {
```

Comment:
Was thinking about the location of this method. Probably it could benefit if it was a part of our [security/strings](https://github.com/TryGhost/Ghost/blob/master/core/server/lib/security/string.js) module and if a replacement would happen around [here](https://github.com/TryGhost/Ghost/blob/master/core/server/lib/security/string.js#L12), as it would be more in line with the logic of that module. Would love a second opinion of this @kirrg001

Reply:
It really doesn't matter right now as I am waiting to co-locate the code.

Reply:
In that case, @dexterleng let's just focus on improvements [here](https://github.com/TryGhost/Ghost/pull/10252#discussion_r241003874) just a bit and will be ready to ship after.

Reply:
@gargol. Done!

Reply:
Thanks @dexterleng :+1: Having another :eyes: :smiley: